test/Timelock.spec.ts

```diff
@@ -0,0 +1,165 @@
+import '@nomiclabs/hardhat-ethers';
+const hre = require('hardhat');
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { AbiCoder } from 'ethers/lib/utils';
+import { RegistryFixture, TimelockFixture } from './shared/fixtures';
+import { Registry, Timelock } from '../typechain';
+
+describe('Timelock.sol', function () {
+  let deployer: any;
+  let deployer2: any;
+  let deployer3: any;
+  let deployer4: any;
+  let registry: Registry;
+  let timelock: Timelock;
```

Comment:
With @AlbyGiaco and @luduvigo we agreed to use Pascal case for contract names, can you change these?